flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -2477,6 +2511,28 @@ public void registerCachedFile(String filePath, String name, boolean executable)
                         name, new DistributedCache.DistributedCacheEntry(filePath, executable)));
     }
 
+    private List<ConfigurationNotAllowedError> checkNotAllowedConfigurations() {
```

Comment:
Is it important that this is a `List`?

Reply:
since the returned errors will only be checked whether it is empty, how about just let this method return the boolean value?